Saucier720-app/src/app/core/interfaces/store.ts

```diff
@@ -1,3 +1,4 @@
 export interface Store {
     Name: string;
+    ZipCode: string;
```

Comment:
Zipcode here? I think this isn't a camel case thing as it's one word

Reply:
it's two words